filebeat/docs/faq.asciidoc

```diff
@@ -4,8 +4,26 @@
 This section contains frequently asked questions about Filebeat. Also check out the
 https://discuss.elastic.co/c/beats/filebeat[Filebeat discussion forum].
 
+[[filebeat-network-volumes]]
 === Can I read log files from network volumes?
 
 We do not recommend reading log files from network volumes. Whenever possible, install Filebeat on the host machine and
 send the log files directly from there. Reading files from network volumes (especially on Windows) can have unexpected side
 effects. For example, changed file identifiers may result in Filebeat reading a log file from scratch again.
+
+// === Why isn’t Filebeat collecting lines from my file?
```

Comment:
1. check if can filebeat send to logstash/elasticsearch (e.g. logstash connection established)
2. ignore_older by default 24h. If file is older, it can not be send. (limit removed in master branch with introduction of close_older)